pennylane_lightning_gpu/src/util/cuda_helpers.hpp

```diff
@@ -10,6 +10,7 @@
 #include <cuComplex.h>
 #include <cublas_v2.h>
 #include <cuda.h>
+#include <cusparse_v2.h>
```

Comment:
Is there a benefit to the v2 AI over the original?

Reply:
no, for the moment. in fact, including  cusparse_v2.h is equivalent to include cusparse.h since no addition api provided in cusparse_v2.h.